### PR TITLE
Update intel nuc install instructions

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update intel nuc install instructions

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex/intel-nuc
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1gb03YbX5ZQ_oO74M36b3ybCPwqq29d8Q17c6KaguW18/edit) and resolve comments

### CODE REVIEW

File: templates/appliance/shared/_steps_nuc_flash_usb.html [hunk 1]
```diff
@@ -5,8 +5,8 @@ <h4 class="p-stepped-list__title">
   </h4>
   <div class="p-stepped-list__content">
     <ol>
-      <li>Download and copy the <a href="/download/desktop">Ubuntu Desktop 20.04 LTS image</a> on the first USB flash drive by following the live USB Ubuntu Desktop tutorial for {% if os=='ubuntu' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>{% elif os=='windows' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>{% else %}<a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a>{% endif %}</li>
-      <li>Copy the Ubuntu Core image for Intel NUC on the second USB flash drive</li>
+      <li>Download and copy the <a href="/download/desktop">Ubuntu Desktop 20.04 LTS image</a> on the first USB flash drive by following the live USB Ubuntu Desktop {% if os=='ubuntu' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">tutorial for Ubuntu</a>{% elif os=='windows' %}<a href="/tutorials/tutorial-create-a-usb-stick-on-windows">tutorial for Windows</a>{% else %}<a href="/tutorials/tutorial-create-a-usb-stick-on-macos">tutorial for macOS</a>{% endif %}</li>
+      <li>Copy the Ubuntu Appliance image you downloaded to get here on to the second USB flash drive</li>
```
Comment (anthonydillon) [inline templates/appliance/shared/_steps_nuc_flash_usb.html:9]: The sentence doesn't make sense. I think its the "to get here on to" part.

File: templates/appliance/shared/_intel-nuc.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <div class="row u-vertically-center">
     <div class="col-8">
       {% if appliance.iso_url.pc %}
-      <h1>Install the Ubuntu {{ appliance.name }} appliance for Intel NUC</h1>
```
Comment (anthonydillon) [inline templates/appliance/shared/_intel-nuc.html:9]: You didn't kill the auto download here

Comment (pmahnke) [reply templates/appliance/shared/_intel-nuc.html:9]: ha, no, I was going to add that as an instruction instead... sorry

File: templates/appliance/shared/_steps_nuc_flash_usb.html (hunk 1 above)
Comment (pmahnke) [reply templates/appliance/shared/_steps_nuc_flash_usb.html]: good catch, I just removed the 'to get here'

**Pull request merged**